src/com/company/Sudoku.java

```diff
@@ -0,0 +1,380 @@
+package com.company;
+
+import javax.swing.*;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class Sudoku {
+    public Sudoku(Scanner gameScan) {
+        this.gameScan = gameScan;
+    }
+
+    private  final Scanner gameScan;
+
+    public  boolean startup() {
+
+        System.out.println("Witamy w Sudoku Zugaj Edition V1\nBy rozpocząć rozgrywkę wpisz:  Sudoku");
+        String x= gameScan.next();
+        return x.equals("Sudoku");
+    }
+
+    public boolean validatePion(int[][] board){
+        for(int z=0;z<=8;z++){
+            for(int i=0;i<=8;i++) {
+                for(int j=0;j<=8;j++){
+                    if(i!=j){
+                        if (board[z][i]==board[z][j]) {
+                            return false;
+                        }
+                    }
+                }
+            }
+        }
+        return true;
+    }
+    public boolean validatePoziom(int[][] board){
+        for(int i=0;i<=8;i++){
+            for(int z=0;z<=8;z++) {
+                for(int j=0;j<=8;j++){
+                    if(z!=j){
+                        if (board[z][i]==board[j][i]) {
+                            return false;
+                        }
+                    }
+                }
+            }
+        }
+        return true;
+    }
+    public boolean validateKwadrat(int[][]board) {
+        //String[] string=new String[9];
+        for (int i = 0; i <= 6; i += 3) {
+            for (int j = 0; j <= 6; j += 3) {
+                int[] helping = {board[i][j], board[i + 1][j], board[i + 2][j], board[i + 1][j], board[i + 1][j + 1], board[i + 1][j + 2], board[i + 2][j], board[i + 2][j + 1], board[i + 2][j + 2]};
+                Arrays.sort(helping);
+                for (int a = 0; a <= 6; a++) {
+                    for (int b = 0; b <= 8; b++) {
+                        if (a != b) {
+                            if (helping[a] == helping[b]) {
+                                return false;
+                            }
+                        }
+                    }
+
+                    //System.out.println(strings[0] +strings[1] + strings[2] + strings[3] + strings[4] + strings[5] + strings[6] + strings[7] + strings[8]);
+
+                }
+            }
+        }return true;
```

Comment:
A tu to już zupełnie nie wiem co się dzieje xD i spróbujemy to trochę uprościć żeby nie trzeba było nowych tabel robić.